src/utils.py

```diff
@@ -73,11 +96,14 @@ def sync_orders(project_id_, project_name):
                 order_data = mp.get_project_item(project_id=project_id_,
                                                  project_item_id=event.project_item_id)
                 wc.create_marketplace_order(project=project_name,
-                                            offering="80340a86eb214b709e03db364be6b7a7",
-                                            # hardcoded offering=Nordic Resource 1
-                                            plan=None,  # None so it is better for testing
-                                            # plan="f8afe3975cfa4dcc9a199b7ccf7c71bb",  # hardcoded plan
-                                            attributes=None,
+                                            offering=wc._get_offering(order_data.attributes.service,
+                                                                      project_name)['uuid'],
+                                            # offering name must match with waldur offering
+                                            plan=wc._get_offering(order_data.attributes.service,
+                                                                  project_name)['plans'][0]['uuid'],
+                                            # 0 is index of the plan
```

Comment:
where is 0?

Reply:
0 is the latest plan of an offering
plan=wc._get_offering(order_data.attributes.service,
                                                                  project_name)['plans']**[0]**['uuid']